src/libsync/theme.cpp

```diff
@@ -228,6 +228,11 @@ QString Theme::helpUrl() const
     return QString::fromLatin1("https://doc.owncloud.org/desktop/%1.%2/").arg(MIRALL_VERSION_MAJOR).arg(MIRALL_VERSION_MINOR);
 }
 
+QString Theme::conflictHelpUrl() const
+{
+    return helpUrl() + QStringLiteral("conflicts.html");
```

Comment:
When helpUrl is empty, this should be empty too.

Reply:
updated